### DISCUSSION TITLE
用户名和密码是什么？

### DISCUSSION THREAD

**Discussion by handsomeallan** [GaiZhenbiao/ChuanhuChatGPT#618] [Q&A]
直接启动程序就会提示用户名和密码，这个是哪里的？

**Comment by Keldos-Li** [comment]
看 [使用教程](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/使用教程)。

**Reply by handsomeallan** [reply]
if __name__ == "__main__":
    reload_javascript()
    auth = auth_list if authflag else None
    demo.queue(concurrency_count=CONCURRENT_COUNT).launch(
        server_name=server_name,
        server_port=server_port,
        share=share,
        auth=auth_list,
        favicon_path="./assets/favicon.ico",
        inbrowser=not dockerflag, # 禁止在docker下开启inbrowser
    )

这里的auth = auth_list有问题，应该是auth = auth。上面判断后赋值给了auth，而非auth_list。应该是这样的吧？

**Reply by Keldos-Li** [reply]
你说的对，刚 c4452490db0ec2bcf554bdd8b28940b818e6b09c 应该解决了，现在应该没问题了。

我们怎么merge pull request的时候又没测试，唉。
